front-end/src/Profile.js

```diff
@@ -137,6 +138,11 @@ const UserInfo = ({ data }) => {
     }
   }, [data])
 
+    const [copied, setCopied] = useState(false);
+    const onCopy = () => {
+	setCopied(true);
+    }
+  //CopyToClipboard must have exactly one child, hence why the button and copied text are wrapped in a div.
```

Comment:
As a heads up, if you want to make comments inside the return statement, you can do them like so:
`{/*Comment goes here*/}`

I think this comment would be better placed above `CopyToClipboard`

Reply:
As an aside though, I'm not sure if it's good practice or not to put comments there